docs/low-code-spark/fabrics/emr-fabric.md

```diff
@@ -0,0 +1,90 @@
+
+<img src={require('./img/livy.png').default} alt="EMR create cluster" width="75%" />
+
+### Allow connectivity between Amazon EMR and Prophecy
+
+To configure the necessary network settings for seamless integration, specific modifications to the security groups of your EMR cluster are required. Please follow the instructions outlined below:
+If you intend to utilize Prophecy as a SaaS (Software as a Service) solution, please note that the Prophecy public IP is `3.133.35.237`. Ensure that the Core security group's outbound rule allows connections to this IP address.
+
+1. Modify the **Primary Node** security group:
+   - Allow incoming connections to port `8998` from the Prophecy IP.
+   - This can be achieved by adding an inbound rule to the Master security group that permits incoming traffic on port `8998` from the Prophecy IP address.
+2. Modify the **Core Node** security group:
+   - Allow outgoing connections to the Prophecy public IP over HTTPS.
+   - To enable this, add an outbound rule to the Core security group that allows outgoing traffic over HTTPS protocol to the Prophecy public IP.
+
+By implementing these adjustments to your EMR cluster's security groups, you will establish the necessary network configuration for effective integration with Prophecy.
+
+## Create a Fabric to connect Prophecy to EMR
+
+Navigate to Prophecy's UI and click on **Create Fabric**. The Fabric will establish a connection with your EMR cluster and utilizes it as the execution engine for your Pipelines.
+<img src={require('./img/create-entity-emr.png').default} alt="EMR create cluster" width="75%" />
+<br/>
+<br/>
+
+**Name** your EMR Fabric and click on **Continue**.
+
+<br/>
+<br/>
+
+<img src={require('./img/name-emr.png').default} alt="EMR create Fabric" width="75%" />
+
+Choose **EMR** as your **Provider**.
+
+<img src={require('./img/emr-f-abric.png').default} alt="EMR create Fabric" width="75%" />
+
+Before proceeding, it is crucial to ensure that all the required settings are properly configured. If you are uncertain about the necessary configurations, we recommend reaching out to your cloud infrastructure team for additional guidance and assistance. They will be able to provide you with the specific information and support needed to ensure a successful setup.
+
+Enter your AWS credentials under **Access Key** and **Secret Key**. Choose the **Region** that your EMR cluster is running in.
+
+**Click** on **Fetch environments**.
+<img src={require('./img/emr-aws-cred.png').default} alt="EMR cred" width="75%" />
+
+**Select** your EMR cluster that you would like to connect to under **Spark Environment**.
+
+<img src={require('./img/emr-cluster-select.png').default} alt="EMR select" width="75%" />
+
+Most of the fields should be automatically populated after selecting a EMR cluster. Enter the S3 path that points to the location where you would like your logs to persist.
```

Comment:
Should we add a note about S3 bucket/path permissions? 
What is the entity that should be enabled to write logs?
EMR cluster - IAM role?
Prophecy cluster - IAM role? 
Individual users ?
Service principals?

Reply:
no changes needed; this is handled by the prophecy token